src/page-multi-facility/FacilityInputForm.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useContext, useState } from "react";
```

Comment:
Mostly this was just a copy/paste from what we already had in the Facility Page, but I needed this component to live within (not aside) from the EpidemicModelProvider so that I could access the current model to grab its inputs for storage.